lib/SILOptimizer/Analysis/AliasAnalysis.cpp

```diff
@@ -744,7 +754,7 @@ bool AliasAnalysis::canBuiltinDecrementRefCount(BuiltinInst *BI, SILValue Ptr) {
     // to be an owned reference and disallows addresses. Conservatively handle
     // address type arguments as and conservatively treat all other values
     // potential owned references.
-    if (Arg->getType().isAddress() || EA->mayReleaseContent(Arg, Ptr))
+    if (Arg->getType().isAddress() || EA->mayReleaseReferenceContent(Arg, Ptr))
```

Comment:
Are we sure we don't have any builtins with indirect arguments?

Reply:
I think it's safe to enable this now for builtins taking addresses